packages/kbn-reporting/common/job_types.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const CSV_JOB_TYPE = 'csv_searchsource';
+
+export const PDF_JOB_TYPE = 'printable_pdf';
+export const PDF_JOB_TYPE_V2 = 'printable_pdf_v2';
+
+export const PNG_JOB_TYPE = 'PNG';
+export const PNG_JOB_TYPE_V2 = 'PNGV2';
```

Comment:
I think this file contains some variables that should stay in the Reporting plugin for now. 

I recommend we rename this file to `packages/kbn-reporting/common/constant.ts` and export only these items:
```
export const CONTENT_TYPE_CSV = 'text/csv';
export const CSV_BOM_CHARS = '\ufeff';
export const UI_SETTINGS_CSV_SEPARATOR = 'csv:separator';
export const UI_SETTINGS_CSV_QUOTE_VALUES = 'csv:quoteValues';
```